okhttp/src/main/java/io/grpc/okhttp/OkHttpTlsUpgrader.java

```diff
@@ -33,6 +33,7 @@
 
 import com.google.common.base.Preconditions;
 
+import com.squareup.okhttp.internal.tls.OkHostnameVerifier;
```

Comment:
things like "internal" frighten me.  Is this going to be stable?  Is there any chance this will cause pain trying to upgrade later?